charts/seed-machines/charts/packet-machineclass/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
Delete this file (the whole `packet-machineclass` (respectively, the whole `seed-machines) directory).

Reply:
Done